pkg/upstream/cluster/loadbalancer.go

```diff
@@ -46,7 +46,7 @@ func init() {
 	}
 	RegisterLBType(types.RoundRobin, rrFactory.newRoundRobinLoadBalancer)
 	RegisterLBType(types.Random, newRandomLoadBalancer)
-	RegisterLBType(types.WeightedRoundRobin, newSmoothWeightedRRLoadBalancer)
+	RegisterLBType(types.WeightedRoundRobin, newWeightedRRLoadBalancer)
```

Comment:
Maybe better that  named `newWeightedWRRLoadBalancer` .

Reply:
I think WeigthtedRR = WRR?

Reply:
ok